.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npm test
```

Comment:
You want to run all tests before each commit?

Reply:
yea im ok with that, what do u think?